RampUpAuto/src/main/java/seleautomation/com/RampUpAuto/App.java

```diff
@@ -1,13 +1,29 @@
 package seleautomation.com.RampUpAuto;
 
-/**
- * Hello world!
- *
- */
-public class App 
-{
-    public static void main( String[] args )
-    {
-        System.out.println( "Hello World!" );
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class App {
+    public static void main(String[] args) {
+        WebDriver driver = null;
+        WebDriverManager.chromedriver().browserVersion("120.0.6099.71").setup();
+        ChromeOptions options = new ChromeOptions();
+        driver = new ChromeDriver(options);
+        driver.get("https://www.google.com/");
+        driver.findElement(By.name("q")).sendKeys("Selenium Tutorials");
+        driver.findElement(By.name("q")).sendKeys(Keys.ENTER);
+
+        if (driver.findElement(By.name("q")).getText().equals("Selenium Tutorials")) {

```

Comment:
`driver.findElement(By.name("q")).getText()` retrieves the context text in the name of the q not the web title , could you please sth like :
` driver.getTitle(); `  we used to verify or assert the title of a web page during testing